using-mobile-extensions/adobe-campaign-standard/adobe-campaignclassic-api-reference.md

```diff
@@ -0,0 +1,70 @@
+# Campaign API reference
+
+## Set linkage fields
+
+Allows the Campaign to connect fields from separate databases to create a more developed and personalized messaging experience for their end user. Profile template based messages that can contain PII based personalization will be downloaded. 
+
+{% tabs %}
+
+{% tab title="iOS" %}
+### setLinkageFields
+
+
+#### Syntax
+
+```objectivec
++ (void) setLinkageFields: (nonnull NSDictionary<NSString*, NSString*>*) linkageFields;
+```
+
+#### Objective-C
+
+The set linkage fields are stored as base64 encoded JSON string in memory and sent in a custom HTTP header 'X-InApp-Auth' in all future Campaign rules download requests until ACPCampaign::resetLinkageFields is invoked.
```

Comment:
Please change the sentence so it reads: "The set linkage fields are stored as a base64-encoded JSON string in memory and sent in a custom HTTP header 'X-InApp-Auth' in all future Campaign rules download requests until ACPCampaign::resetLinkageFields is invoked."

Is ""set linkage" the exact field title? If yes, it should be in bold and init caps like this: "**Set linkage**".